### PR TITLE
rehibernate

### PR DESCRIPTION
**Pull request by MartinFrac**
refactor relacji i update skryptow

### CODE REVIEW

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 
+import com.infoshareacademy.tailandczycy.model.Budget;
+import com.infoshareacademy.tailandczycy.model.Category;
+import com.infoshareacademy.tailandczycy.model.Expense;
 import freemarker.template.Template;
 import freemarker.template.TemplateException;
 
 import javax.inject.Inject;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'javax.servlet.ServletException'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.math.BigDecimal;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'java.math.BigDecimal'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
+import java.time.LocalDate;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'java.time.LocalDate'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 4]
```diff
@@ -1,16 +1,26 @@
 package com.infoshareacademy.tailandczycy.web;
 
+import com.infoshareacademy.tailandczycy.dao.BudgetDao;
+import com.infoshareacademy.tailandczycy.dao.CategoryDao;
+import com.infoshareacademy.tailandczycy.dao.ExpenseDao;
 import com.infoshareacademy.tailandczycy.freemarker.TemplateProvider;
 
+import com.infoshareacademy.tailandczycy.model.Budget;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'com.infoshareacademy.tailandczycy.model.Budget'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 5]
```diff
@@ -1,16 +1,26 @@
...
+import com.infoshareacademy.tailandczycy.model.Category;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'com.infoshareacademy.tailandczycy.model.Category'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 6]
```diff
@@ -1,16 +1,26 @@
...
+import com.infoshareacademy.tailandczycy.model.Expense;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'com.infoshareacademy.tailandczycy.model.Expense'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java [hunk 7]
```diff
@@ -1,16 +1,26 @@
...
 import freemarker.template.Template;
 import freemarker.template.TemplateException;
 
 import javax.inject.Inject;
+import javax.servlet.ServletConfig;
```
Comment (isacademyci) [inline finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: MINOR Remove this unused import 'javax.servlet.ServletConfig'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by isacademyci** [comment]
SonarQube analysis reported 7 issues
* ![MINOR][MINOR] 7 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 4 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 5 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 6 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 1 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 2 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java (hunk 3 above)
Comment (MichalNowakowski) [reply finance-manager-web/src/main/java/com/infoshareacademy/tailandczycy/web/MainPageServlet.java]: usun pls

File: finance-manager-web/src/main/resources/META-INF/persistence.xml [hunk 8]
```diff
@@ -6,6 +6,7 @@
     <property name="hibernate.dialect" value="org.hibernate.dialect.MySQL5Dialect"/>
     <property name="hibernate.show_sql" value="true"/>
     <property name="hibernate.format_sql" value="true"/>
+    <property name="hibernate.hbm2ddl.auto" value="create"/>
```
Comment (MichalNowakowski) [inline finance-manager-web/src/main/resources/META-INF/persistence.xml]: nie mialo byc validate jak macie import z sql?

**Pull request merged**